### PR TITLE
handle the case where release note not on same line as tag

### LINKED ISSUE #2054: Release Note Auto-Assign Doesn't Recognize If RN On Next Line
```
noteMatcher := regexp.MustCompile("Release note.*```(.+)```")
```

The wildcard matcher does not recognize newlines.  Need to set the (?s) flag.

### PR DESCRIPTION
**Pull request by grodrigues3**
The (?s) flag allows the wildcard operator to match newlines as well.

fixes #2054

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/2056)

### CODE REVIEW

File: mungegithub/mungers/release-note-label_test.go [hunk 1]
```diff
@@ -249,6 +249,11 @@ func TestGetReleaseNote(t *testing.T) {
 			expectedReleaseNoteVariable: releaseNoteNone,
 		},
 		{
+			body:                        "**Release note**:\n\n<other unimportant information> ```NONE```",
```
Comment (apelisse) [inline mungegithub/mungers/release-note-label_test.go]: isn't it weird that we skip "<other unimportant information>". What if it was important? (like being something completely different?). The .* becomes VERY lenient now that you have the `(?s)` modifier.

Should we use:
`Release Note:\s*BQ BQ BQ`.....

Comment (grodrigues3) [inline mungegithub/mungers/release-note-label_test.go]: also, check out the PR template https://github.com/kubernetes/kubernetes/edit/master/.github/PULL_REQUEST_TEMPLATE.md 

The goal of the regex is to match what's there explicitly.

Comment (grodrigues3) [inline mungegithub/mungers/release-note-label_test.go]: We ask specifically n the release-note template for the release note to be between three ticks.
E.g. ```Release Note Goes Here```

File: mungegithub/mungers/release-note-label_test.go [hunk 2]
```diff
@@ -244,17 +244,22 @@ func TestGetReleaseNote(t *testing.T) {
...
+			expectedReleaseNoteVariable: releaseNoteNone,
+		},
+		{
+			body:                        "**Release note**:\n\n  ```This is a description of my feature```",
 			expectedReleaseNote:         "This is a description of my feature",
 			expectedReleaseNoteVariable: releaseNote,
 		},
 		{
-			body:                        "**Release note**: <other unimportant information> ```This is my feature. There is some action required for my feature.```",
+			body:                        "**Release note**: ```This is my feature. There is some action required for my feature.```",
```
Comment (apelisse) [inline mungegithub/mungers/release-note-label_test.go]: can we add a test that leads to no release note? One that shows that the regex is less lenient?

**Comment by apelisse** [comment]
Looks good

**Pull request merged**